timesketch/api/v1/resources.py

```diff
@@ -695,10 +700,140 @@ def post(self, sketch_id):
 class AggregationResource(ResourceMixin, Resource):
     """Resource to query for aggregated results."""
 
+    @login_required
+    def get(self, sketch_id, aggregation_id):  # pylint: disable=unused-argument
+        """Handles POST request to the resource.
+
+        Handler for /api/v1/sketches/:sketch_id/aggregation/:aggregation_id
+
+        Args:
+            sketch_id: Integer primary key for a sketch database model
+            aggregation_id: Integer primary key for an agregation database model
+
+        Returns:
+            JSON with aggregation results
+        """
+        # TODO: Implement once aggregations are saved in the datastore.
+        return {}
+
+    @login_required
+    def post(self, sketch_id, aggregation_id):  # pylint: disable=unused-argument
+        """Handles POST request to the resource.
+
+        Handler for /api/v1/sketches/:sketch_id/aggregation/:aggregation_id
+
+        Args:
+            sketch_id: Integer primary key for a sketch database model
+            aggregation_id: Integer primary key for an agregation database model
+        """
+        # TODO: Implement once we have an aggregation model in the datastore.
+        abort(HTTP_STATUS_CODE_FORBIDDEN)
+
+
+class AggregationExploreResource(ResourceMixin, Resource):
+    """Resource to send an aggregation request."""
+
+    RESULT_FIELDS = frozenset(['_shards', 'hits', 'timed_out', 'took'])
```

Comment:
It looks like you are filtering these out? In that case, rename to somthing like:
```REMOVE_FIELDS```

Reply:
ack